tutorial/OpenTelemetry/OpenTelemetry_OPEA_Guide.rst

```diff
@@ -0,0 +1,178 @@
+.. _OpenTelemetry_OPEA_Guide:
+
+OpenTelemetry on OPEA Gudie
+#############################
+
+Overview
+********
```

Comment:
Improper heading format: used * instead of ##, should use standard Markdown heading syntax #, ##, ###.

Reply:
RST format seems to also use * .  https://sublime-and-sphinx-guide.readthedocs.io/en/latest/topics.html